src/ImageSharp/ImageFrame.cs

```diff
@@ -13,37 +13,28 @@ namespace SixLabors.ImageSharp
     /// In case of animated formats like gif, it contains the single frame in a animation.
     /// In all other cases it is the only frame of the image.
     /// </summary>
-    public abstract partial class ImageFrame : IDisposable
+    public abstract partial class ImageFrame : IConfigurable, IDisposable
```

Comment:
This definition somewhat implies that a frame is configurable separatly from it's owner image.
Shouldn't we rename `IConfigurable` to something like `IConfigurationProvider`?

Reply:
Happy to do so. It's an internal interface anyway so not a breaking change.